### DISCUSSION TITLE
Scrolling of add-labels list during issue creation is broken on Android app

### DISCUSSION THREAD

**Discussion by bskinn** [community/community#107546] [Mobile]
When I try to add labels to a new issue I'm creating, I can open the Labels view and choose labels to add.

However, the scrolling is broken: I can scroll freely as long as I keep my finger down, but after that I can only scroll down from where I stopped.

Once I lift my finger to stop scrolling, if I try to scroll back up I get the 'refresh page' pull-down behavior. In order to see the labels higher in the list, I have to exit the 'add label' interface and re-enter.

Android GitHub app v1.148.0
Pixel 6
Android v14, build UQ1A.240105.002

**Comment by alcere** [comment]
@bskinn We apologize for the trouble here. We have created an internal issue to resolve this as quickly as possible. Thank you for bringing this to our attention!

**Reply by bskinn** [reply]
No worries, thanks for the speedy reply! And, thanks for all the work on the app & other tools.

**Reply by archon810** [reply]
Thank you, I just ran into the same issue and was about to file a ticket but found this thread.

**Comment by dileepa-dev** [comment]
same here
